app/views/content/returning-to-teaching.md

```diff
@@ -165,6 +155,7 @@ If you’re returning to teaching, you might want to re-familiarise yourself wit
 #### Other things to consider
 
 * attend an [online return to teaching event](/event-categories/online-q-as)
+* return to the classroom on a temporary basis by joining a [supply teaching agency](https://www.gov.uk/government/publications/international-qualified-teacher-status-teachers-standards)
```

Comment:
```suggestion
* return to the classroom by [registering with a supply teaching agency](/urgent-call-for-qualified-teachers)
```

Reply:
I wonder if we should put this as the last bullet - we would rather they considered returning in full rather than supply?